client/web/src/enterprise/repo/settings/RepoSettingsPermissionsPage.tsx

```diff
@@ -132,9 +132,13 @@ export const RepoSettingsPermissionsPage: FC<RepoSettingsPermissionsPageProps> =
                                 </tr>
                                 <tr>
                                     <th>Last incremental sync</th>
-                                    <td>
-                                        <Timestamp date={permissionsInfo.updatedAt} />
-                                    </td>
+                                    {permissionsInfo.updatedAt === null ? (
+                                        <td>Never</td>
+                                    ) : (
+                                        <td>
+                                            <Timestamp date={permissionsInfo.updatedAt} />
+                                        </td>
+                                    )}
```

Comment:
I think we can move `<td` outside the ternary condition:
```suggestion
                                   <td>
                                   {permissionsInfo.updatedAt === null ? 'Never' : <Timestamp date={permissionsInfo.updatedAt} />}
                                   </td>
```